<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompetenceManager</class>
 <widget class="QDialog" name="CompetenceManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1321</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestionnaire des comp√©tences</string>
  </property>
  <layout class="QHBoxLayout" name="layout_main" stretch="0">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="renderHints">
        <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::TextAntialiasing</set>
       </property>
       <property name="viewportUpdateMode">
        <enum>QGraphicsView::FullViewportUpdate</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layout_buttons"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
